frontend/src/components/SystemsTable/SystemTableTools.tsx

```diff
@@ -38,7 +38,7 @@ interface Props {
   onChange: (value: Partial<Filter>) => void;
   metricOptions: string[];
   selectedSystemIDs: string[];
-  setActiveSystemIDs: React.Dispatch<React.SetStateAction<string[]>>;
+  setActiveSystemIDs: (ids: string[]) => void;
```

Comment:
Thanks for fixing the type annotation! Could you rename it to `onActiveSystemChange` so it conforms to the naming conventions of event handlers?